src/cow-react/modules/limitOrders/containers/TradeButtons/limitOrdersTradeButtonsMap.tsx

```diff
@@ -129,11 +129,11 @@ export const limitOrdersTradeButtonsMap: { [key in LimitOrdersFormState]: Button
     text: 'Sell amount is too small',
   },
   [LimitOrdersFormState.QuoteError]: ({ quote }: TradeButtonsParams) => {
+    const defaultError = quoteErrorTexts[GpQuoteErrorCodes.UNHANDLED_ERROR]
+
     return (
       <SwapButton disabled={true}>
-        <Trans>
-          {quote.error ? quoteErrorTexts[quote.error.type] || 'Error loading price. Try again later.' : 'Unknown error'}
-        </Trans>
+        <Trans>{quote.error ? quoteErrorTexts[quote.error.type] || defaultError : defaultError}</Trans>
```

Comment:
What about:

Warning, code not tested!

```suggestion
        <Trans>{(quote.error && quoteErrorTexts[quote.error.type]) || defaultError }</Trans>
```